### ISSUE TITLE
Write a spec file

### ISSUE THREAD

**Issue by ghost** [paolorotolo/spotify-linux-installer#5]
> write a spec file to run the script and set up a repo like https://github.com/aspiers/opensuse-spotify-installer

**Comment by ghost** [comment]
I've forked the repo and will work on this

**Comment by ghost** [comment]
I've got the spec file finished for this (sorry it took longer than expected).

**Comment by paolorotolo** [comment]
@jonlandrum Awesome, can you open a Pull Request please?
We'll close this when the pull is merged.

**Comment by ghost** [comment]
@paolorotolo certainly. I fell asleep last night trying to read about building an RPM to make sure I've got the spec file right, sorry for not opening the pull request earlier.

**Comment by ghost** [comment]
Also, you'll notice line 9 is commented out: `#Requires:   binutils`. That's the part I wanted to test. It appears RPM is very good at determining dependencies, but I haven't been able to confirm if it can pick up the binutils dependency or not.
